velox/exec/CallbackSink.h

```diff
@@ -30,6 +31,11 @@ class CallbackSink : public Operator {
         callback_{callback} {}
 
   void addInput(RowVectorPtr input) override {
+    EvalCtx evalCtx(operatorCtx_->execCtx(), nullptr, input.get());
```

Comment:
EvalCtx is supposed to be used only in expression evaluation. It looks like ensureFieldLoaded is generally useful and therefore needs to be moved into a different place, perhaps, velox/exec/OperatorUtils.h.

Reply:
As discussed, this logic can be moved into a static method on LazyVector.